### PR TITLE
Add blog on cycle 6 results

### PR DESCRIPTION
**Pull request by m52go**

### CODE REVIEW

File: _posts/2019-11-01-cycle-6-results.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Bisq DAO Cycle 6: Results"
+author: Steve Jain
+excerpt: "Cycle 6 of the Bisq DAO ended at block 595 146 on October 17 2019. This post covers its results. <br><br>"
```
Comment (devinbileck) [inline _posts/2019-11-01-cycle-6-results.md]: It ended at block 599826.

File: _posts/2019-11-01-cycle-6-results.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ref: blog
+---
+
+Cycle 6 ended a few days ago. This post summarizes its results.
```
Comment (devinbileck) [inline _posts/2019-11-01-cycle-6-results.md]: It ended a few weeks ago now.

**Review by devinbileck — CHANGES_REQUESTED** [review]
Just a couple minor corrections.

**Comment by m52go** [comment]
Addressed both items and squashed to one commit. Thanks Devin!

**Pull request merged**